addons/l10n_lt/__manifest__.py

```diff
@@ -0,0 +1,29 @@
+# Author: Silvija Butko. Copyright: JSC Focusate.
+{
+    'name': "Lithuania - Accounting",
+    'version': '1.0.0',
+    'summary': 'accounting, Lithuanian',
+    'license': 'LGPL-3',
+    'author': "Focusate",
+    'website': "http://www.focusate.eu",
+    'category': 'Localization',
+    'depends': [
+        'l10n_multilang',
```

Comment:
could you add `account` as well ?

Reply:
Sure, though isnt `l10n_multilang` not enough? Cause that one depends on `account`?

Reply:
you're right, no need to put `account` here

Reply:
@oerp-odoo Of course, English makes it easier in an international context, but don't you need to report in Lithuanian anyways?  (as for us, it is ok to put everything in Lithuanian, but this can even be better)

Reply:
> @oerp-odoo Of course, English makes it easier in an international context, but don't you need to report in Lithuanian anyways? (as for us, it is ok to put everything in Lithuanian, but this can even be better)

which part do you mean? All data that is translatable, is translated in Lithuania. So it is still reported in native Language.

Reply:
Multilang is mostly when a country has multiple official languages and sometimes to make it clear for non-residents, but other localization would directly do the accounts,... in Lithuanian.

Reply:
Fine the way it is now.  Just that you know.

Reply:
I understand, though we made it multilang (and we do for all our modules, be it localization or not. Except maybe in some rare cases where there is no point to translate some data), so it would be more user friendly in multicompany environment. For example, if there are multiple companies with users speaking different languages and they need to at least have some understanding about other companies COA, taxes etc. 
So I think it is a good idea to provide English language there. For native people, there are translations for reports anyway. In my opinion it is win-win:)